spec/classes/init_spec.rb

```diff
@@ -25,7 +25,17 @@
 
     it { is_expected.to contain_file('/etc/condor/config.d/docker') }
 
-    it { is_expected.to contain_file('/etc/condor/config.d/schedd').with_content(%r{^DAEMON_LIST = MASTER, SCHEDD}) }
+    it do
+      is_expected.to contain_file('/etc/condor/config.d/schedd').with(
```

Comment:
Spec is needed for the reason ordering.  This can only be checked in rspec-puppet (it can't be done via rspec-beaker).

Reply:
sorry @jhoblitt, I don't get what's wrong, can you explain a bit further, please? is there something wrong with the order? 🤔

Reply:
i think it is a bit better 👍🏻